CONTRIBUTING.md

```diff
@@ -121,15 +121,20 @@ The full Meilisearch release process is described in [this guide](https://github
 Depending on the developed feature, you might need to provide a prototyped version of Meilisearch to make it easier to test by the users.
 
 The prototype name must follow this convention: `prototype-X-Y` where
-- `X` is the feature name formatted in `kebab-case`
+- `X` is the feature name formatted in `kebab-case`. It should not end with a number segment.
 - `Y` is the version of the prototype, starting from `0`.
 
-Example: `prototype-auto-resize-0`.
+✅ Example: `prototype-auto-resize-0`.
+
+❌ Bad example: `auto-resize-0`: lacks the `prototype` prefix.
+❌ Bad example: `prototype-auto-resize`: lacks the version suffix.
+❌ Bad example: `prototype-auto-resize-0-0`: feature name ends with a number segment.
```

Comment:
I think you want to put `<br>`

<img width="1096" alt="Capture d’écran 2023-02-09 à 17 58 24" src="https://user-images.githubusercontent.com/20380692/217884453-24413990-f40e-46e6-a93c-30eb7199fb57.png">

Reply:
Added the brs